### ISSUE TITLE
Esc and Alt-Click not work

### ISSUE THREAD

**Issue by VladimirKalachikhin** [ppete2/Leaflet.PolylineMeasure#45]
Actions by Esc key and Alt-Click not work for me.
Ctrl-Click are work.

Firefox 62.0.3 

Esc:
ReferenceError: resumeFirstpointFlag is not defined[Learn More] Leaflet.PolylineMeasure.js:516:1

Alt-Click:
silently

**Comment by ppete2** [comment]
I could reproduce the problem with ESC-key and solved it.
But I could not reproduce problems with ALT-click: points are being removed without causing an error message on the JS-console.

**Comment by VladimirKalachikhin** [comment]
I have Alt-Click problem in a Firefox safe mode too - without addons.

What information can I provide?

**Comment by VladimirKalachikhin** [comment]
Alt-Click are move a window in Ubuntu MATE. I use Ubuntu MATE.

**Comment by ppete2** [comment]
Could you please describe exactly each step you are doing with mouse and keyboard? And what exactly doesn't work as expected? 
Are you able to test it on a friends's computer with another operating system but with firefox too: still the same problem?

**Comment by VladimirKalachikhin** [comment]
Of course, I have Alt-Click problem in "standard" mode of Firefox. I turn it to safe mode to avoid affect of addons.
Another operating system? Windows? It`s no so easy... I have a Windows Vista with Firefox 52.9.0

So, I opening https://ppete2.github.io/Leaflet.PolylineMeasure/demo1.html in Firefox under Windows on one laptop, and under Ubuntu on another:
Open page - ok
Click to |-> - ok
Draw line with some number of points - ok
Esc to stop drawing - ok
After - I pointed a cursor to last point, and pressing Alt and Click.
Windows - deleting last point - ok
Ubuntu - nothing.

Also, code 
```
            // not just used for dragging Cirles but also for deleting circles and resuming line at its starting point.
        _dragCircle: function (e1) {
            if (e1.originalEvent.ctrlKey) {   // if user wants to resume drawing a line
            	alert("Ctrl key pressed!");
```
works under Windows (fires alert), but not under Ubuntu. I think it's because Alt-Click - global (Ubuntu operating system) hotkey for dragging window.

**Comment by ppete2** [comment]
I just uploaded a new, bugfixed version. Before it wasn't able to resume a line after its last point have been deleted. 
Could you please try it again. But I think this wouldn't fix your problem with the ALT-key in Linux. Do you have an idea what key-combination to be use instead which works on each operating system?

**Comment by ppete2** [comment]
I subsituted ALT-Key with SHIFT-key to remove points. This should work on every operating system now. Does it?
The double click inputs etc. always made troubles on some systems or Browsers in the past, so I resigned from using them. And i want to note that deleting points and resuming a line are just intended to be used on desktop systems with a keyboard. On mobile systems just the basic functionallity of drawing lines by tipping on the screen is intended to work on purpose.
